portfolio/index.html

```diff
@@ -163,7 +163,7 @@ <h3>Consultant site</h3>
         </section>
         <section class="contact-form-section" id="contact">
             <h1>Contact Me</h1>
-            <form class="contact-form" method="POST" data-netlify="true" action="/portfolio/thankyou.html">
+            <form class="contact-form" method="GET" data-netlify="true" action="thankyou.html" >
```

Comment:
Maybe the article about http methods will be useful https://www.w3schools.com/tags/ref_httpmethods.asp.
A rough summary. If you send data, it's POST. If you request data it's GET.

Reply:
Doesn't mean that you have to return it back to POST.
Just a useful information to know.

Reply:
yes firstly I use the post method but it give an error "thankyou page not found" then I change it to Get method then it works

Reply:
That's right. Normally your action would be a server url accepting the contact message. The server would store it and would respond with a new url to redirect the user to. Now you don't have the server yet. So it makes sense for now to direct the form to the next page directly.

Reply:
No worries. We will learn how to create a server and do all this storing/redirecting.